app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -26,7 +26,17 @@ angular.module("openshiftConsole")
         }];
         scope.claim.selectedLabels = [];
         DataService.list({group: 'storage.k8s.io', resource: 'storageclasses'}, {}, function(storageClasses) {
-          scope.storageClasses = storageClasses.by('metadata.name');
+          var noclass = {
+            metadata: {
+              name: "No Storage Class",
+              labels: {},
+              annotations: {
+                description: "No storage class will be assigned unless a default class has been assigned by the system administrator."
+                }
+            }
+          };
```

Comment:
Yeah it's a quirk with ui-select. There is an issue in their repo to add this, but it's not supported natively in their widget. We solved it in osc-secrets.html by adding an `x` beside the field to clear, although it made more sense there because you could have more than one secret.

I'm OK with "No Storage Class", but we might want to change the label to "Use Default Storage Class" when there is a default.